modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -232,7 +232,7 @@ image::x12-edi/x12-edi-general.jpg[General tab properties]
 The general tab lets you configure settings for reading and writing X12 messages:
```

Comment:
```suggestion
The general tab enables you to configure settings for reading and writing X12 messages:
```
...and then reword the following list items so that they are PARALLEL: all settings (noun-based) or all user actions (verb-based).
Edits to existing list items are for reference only; I'd like to see the list once more after you make it parallel.
If you don't have time for that, then the edits will at least be a step in the right direction!  :-)

Reply:
GLOBALLY remove periods at the ends of list fragments and move subsequent sentence or sentences to a new para below the list item, as shown in the green edit below: